map_org_names.yaml

```diff
@@ -211,6 +213,7 @@ mappings:
   - ['^[[:space:]]*align[[:space:]]technology[[:space:]]*$','Align Technology, Inc.']
   - ['^[[:space:]]*alion[[:space:]]science[[:space:]]and[[:space:]]technology[[:space:]]*$','Alion Science and Technology Corporation']
   - ['^[[:space:]]*(alipay\\.?com)[[:space:]]*$','Alipay']
+  - ['^[[:space:]]*aliste[[:space:]]*$',"ALISTE ConstrucciÃ³ i gestiÃ³ d'actius Inmobiliaris SL"]
```

Comment:
Is this `Ã³` expected?

Reply:
@lukaszgryglicki yes, the names (with encoding) are how SFDC has it. I get the names by querying the org service

Reply:
Yes but this looks awful, maybe the service returns then wrong - I would rather convert to a correct name with a correct encoding (for example google for that name).
Example (ggogle): `ALISTE ConstrucciÃ³ i gestiÃ³ d'actius Inmobiliaris` -> `A²/Aliste-Construccions i Gestió d'Actius Immobiliaris`
Also what @1010sachin & @fayazg think about this?

Reply:
@lukaszgryglicki yeah, i agree but AFAIK, the affiliations service depends on our org names being exactly the same as what is in their database hence why I use their values as my source of truth

Reply:
OK then, I'm changing to approved.